xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -203,6 +203,20 @@ is_brick_graceful_cleanup_enabled(dict_t *opts)
     return ret ? _gf_false : enabled;
 }
 
+static gf_boolean_t
+gd_has_remote_address(glusterd_conf_t *priv, const char *hostname)
```

Comment:
This function can also be merged with `gd_has_local_address()` and rename it to something like `gd_has_address()` which should also receive a pointer to a list of hosts instead of `priv`.